syntax = "proto3";
package protocol;

service GeneratorService{
  rpc CallRemoteMethod(stream MsgPickleDumps) returns (stream MsgPickleDumps);
}

message MsgPickleDumps{
  bytes content = 1;
  optional bool is_chunked = 2;
}